content/academy/optimizing_scrapers.md

```diff
@@ -0,0 +1,39 @@
+---
+title: How to optimize and speed up your web scraper
+description: We all want our scrapers to run as fast as possible and cost as little money as possible. Learn how to think about performance in the context of web-scraping and automation.
+menuWeight: 18
+category: tutorials
+paths:
+    - optimizing-scrapers
+---
```

Comment:
Maybe also to Web Scraping for beginners?

Reply:
Same as above